ros2bag/ros2bag/verb/play.py

```diff
@@ -83,20 +86,25 @@ def main(self, *, args):  # noqa: D102
         if args.storage_config_file:
             storage_config_file = args.storage_config_file.name
 
-        # NOTE(hidmic): in merged install workspaces on Windows, Python entrypoint lookups
-        #               combined with constrained environments (as imposed by colcon test)
-        #               may result in DLL loading failures when attempting to import a C
-        #               extension. Therefore, do not import rosbag2_transport at the module
-        #               level but on demand, right before first use.
-        from rosbag2_transport import rosbag2_transport_py
-        rosbag2_transport_py.play(
+        topic_remapping = ['--ros-args']
+        for remap_rule in args.remap:
+            topic_remapping.append('--remap')
+            topic_remapping.append(remap_rule)
+
+        storage_options = StorageOptions(
             uri=args.bag_file,
             storage_id=args.storage,
-            node_prefix=NODE_NAME_PREFIX,
+            storage_config_uri=storage_config_file,
```

Comment:
Havnt looked into `StorageOptions` but an empty string is alright here, I assume

Reply:
Yep! Given that the field is a string, there isn't another option for omitting